ics/event.py

```diff
@@ -217,27 +242,49 @@ def all_day(self):
         # the event may have an end, also given in 'day' precision
         return self._begin_precision == 'day'
 
+    def cancel_all_day(self):
```

Comment:
I'm not a fan of the naming, maybe doing `make_all_day(False)` like you suggested could be better